### ISSUE TITLE
[Feature Request[ Add symbolic links for cheat file locations

### ISSUE THREAD

**Issue by OdinVex** [dragoonDorise/EmuDeck#278]
I'd fork and create a pull request, but I'm a bit busy at the moment, and I'm not sure if the project would prefer a different way of handling things besides assuming to use `linkToSaveFolder` to create symbolic links for cheats, such as `linkToSaveFolder pcsx2 cheats ~/.var/app/net.pcsx2.PCSX2/config/PCSX2/cheats`. Thoughts? Would prefer if files could be kept on my SD and loaded from SD instead of flatpak so that my own pnach files don't get lost on flatpak witchery.

**Comment by HammerOfDaGods** [comment]
I was about to request the same, but saw this already here. If it hasn't been done by now I don't think it will be for a while unfortunately.

**Comment by dragoonDorise** [comment]
@OdinVex I might be wrong, but the default for GameBoy Games is same as physical:

<img src="https://i.imgur.com/uVMxON0.jpg" width="300"/>

the Steam Deck A is the Gameboy B and the Steam Deck B is the GameBoy A

**Comment by dragoonDorise** [comment]
@OdinVex yeah, the current RetroArch layout is kinda weird, I also prefer your layout. We already have a setting to tweak Nintendo systems to A=A or A=B, I'd just add GC, GBC and GBA so the layout also changes
